modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -169,6 +176,8 @@ Requests by Client IP:: The sum of requests in the period, grouped by client IP
 * If you select a time range within the last day, data is aggregated in ten minute increments.
 * If you select a time range that is larger than one day, data is aggregated in one hour increments.
 
+Requests by Location: The sum of requests in the period, grouped by country. Data retention for the Requests by Location chart is one hour. 
```

Comment:
Ditto same question about the use of "sum" here rather than total number.
Also, I wonder if it would be more clear to reorder the wording to something like...:
```suggestion
Requests by Location: The total number of requests for each country during the selected time period. Data retention for the Requests by Location chart is one hour. 
```
Is the second sentence necessary, or does it cloud the meaning of the first? What happens after one hour has passed? What displays then?

Reply:
We use the "sum" wording since that is how the query that spits out the values is written.
I would have to play with the suggested wording a little since it's the total number of requests firsts and then country grouping. 

re: data retention, the data is collected continuously over an hour and changes over that time. I think what's displayed changes as the input changes.

Reply:
Oop, I could have just directed you to this note up top: 
Charts displaying the `Last 1 hour` label show data that has been collected during the past 60 minutes, starting with the current time.
Does this make sense?